src/app/components/Search/Search.jsx

```diff
@@ -102,63 +112,50 @@ class Search extends React.Component {
       page: '1',
     });
 
+    this.props.updateSearchKeywords(searchKeywords);
+    this.props.updateField(field);
+
     this.props.router.push(`${appConfig.baseUrl}/search?${apiQuery}`);
   }
 
   render() {
     return (
-      <form
+      <SearchBar
         id="mainContent"
```

Comment:
This should be different because, on the homepage, there are two `mainContent` ids.

Reply:
I'm leaning toward keeping it here and removing it from `Home`